src/NzbDrone.Core.Test/IndexerTests/TorrentRssIndexerTests/TestTorrentRssIndexer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
```

Comment:
there should be resharper settings checked in and loaded automatically, but I'm not sure if stuff like this are gonna be handled automatically, I think they only run if you explicitly call clean up.

https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.sln.DotSettings